base/shell/explorer/taskswnd.cpp

```diff
@@ -1373,20 +1373,19 @@ class CTaskSwitchWnd :
 
     BOOL CALLBACK EnumWindowsProc(IN HWND hWnd)
     {
-        /* Only show windows that still exist and are visible and none of explorer's
-        special windows (such as the desktop or the tray window) */
-        if (::IsWindow(hWnd) && ::IsWindowVisible(hWnd) &&
-            !m_Tray->IsSpecialHWND(hWnd))
-        {
-            DWORD exStyle = ::GetWindowLong(hWnd, GWL_EXSTYLE);
-            /* Don't list popup windows and also no tool windows */
-            if ((::GetWindow(hWnd, GW_OWNER) == NULL || exStyle & WS_EX_APPWINDOW) &&
-                !(exStyle & WS_EX_TOOLWINDOW))
-            {
-                TRACE("Adding task for %p...\n", hWnd);
-                AddTask(hWnd);
-            }
+        if (!::IsWindowVisible(hWnd) || m_Tray->IsSpecialHWND(hWnd))
```

Comment:
Is there any logic change in this block? It seems to just be refactoring, which really needs to happen as a separate commit; otherwise this is really hard to review.

Reply:
taskswnd.cpp was reverted.